eats_worm/Extractor.py

```diff
@@ -442,7 +447,7 @@ def __init__(self, *args, **kwargs):
             print('did not pass root directory')
             return 0
 
-        self.output_dir = os.path.join(kwargs.get('output_dir', self.root), 'extractor-objects')
+        self.output_dir = os.path.join(kwargs.get('output_dir', self.root), 'peakfinding_output')
```

Comment:
We need to think carefully before making changes like this. I think this will break loading for analyses done with older versions of the repo